morphic.js

```diff
@@ -2690,9 +2689,7 @@ Morph.prototype.drawOn = function (aCanvas, aRect) {
 
 Morph.prototype.fullDrawOn = function (aCanvas, aRect) {
     var rectangle;
-    if (!this.isVisible) {
-        return null;
-    }
+    if (!this.isVisible) {return; }
```

Comment:
That should probably be either as removed or all on one line without brackets, i.e.

```
if (!this.isVisible) {
    return null;
}

if (!this.isVisible) return null;
```

Also the code you added doesn't work the same way as the code you removed, because `null !== undefined`.